pkgs/cronet_http/example/lib/main.dart

```diff
@@ -5,21 +5,31 @@
 import 'dart:convert';
 import 'dart:io';
 
-import 'package:cached_network_image/cached_network_image.dart';
 import 'package:cronet_http/cronet_http.dart';
 import 'package:flutter/material.dart';
 import 'package:http/http.dart';
+import 'package:http/io_client.dart';
+import 'package:http_image_provider/http_image_provider.dart';
+import 'package:provider/provider.dart';
 
 import 'book.dart';
 
 void main() {
-  var clientFactory = Client.new; // Constructs the default client.
+  late Client httpClient;
```

Comment:
Do you get an error without the `late`? I would expect `Client httpClient;` or `final Client httpClient;` to work because of the definite assignment.

```suggestion
  final Client httpClient;
```